platforms/documentation/docs/src/docs/userguide/best-practices/best_practices.adoc

```diff
@@ -48,6 +48,11 @@ The items in this guide are meant to be _brief_, _understandable_, and _actionab
 - <<best_practices_dependencies.adoc#best_practices_for_dependencies,Best Practices for Dependencies>>
 - <<best_practices_tasks.adoc#best_practices_for_tasks,Best Practices for Tasks>>
 
-== Further Reference
+== Recommendations
```

Comment:
```suggestion
== Other Advice
```

Should we broaden this title so we can potentially include other related things that aren't really simple recommendations here?  I'm thinking DPEU or blog posts?

Reply:
Sounds good!